src/index.html

```diff
@@ -82,19 +82,16 @@ <h1 id="dateAnnouncement">
                 <div class="modal-content">
                     <div class="modal-header">
                         <h5 class="modal-title" id="exampleModalLabel">Get Vowel Sounds, Encrypted</h5>
-                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
```

Comment:
Removed this, beause we already have a dedicated "close" button, and this  extra button looked off-theme color-wise.